gccadmin (1):
      Daily bump.

ian (1):
      Backport from trunk:

meissner (1):
      setup branch

rguenth (1):
      2019-05-03  Richard Biener  <rguenther@suse.de>

diff --git a/ChangeLog.ibm b/ChangeLog.ibm
new file mode 100644
index 00000000000..d44898dd41b
--- /dev/null
+++ b/ChangeLog.ibm
@@ -0,0 +1,4 @@
+2019-05-03   Michael Meissner  <meissner@linux.ibm.com>
+
+	Clone branch subversion id 270579
+
diff --git a/gcc/ChangeLog b/gcc/ChangeLog
index e9a3fa34129..fe1b698c195 100644
--- a/gcc/ChangeLog
+++ b/gcc/ChangeLog
@@ -1,3 +1,9 @@
+2019-05-03  Richard Biener  <rguenther@suse.de>
+
+	PR tree-optimization/90316
+	* tree-ssa-pre.c (pass_pre::execute): Re-compute DOM fast queries
+	before running VN.
+
 2019-05-03  Jakub Jelinek  <jakub@redhat.com>
 
 	* BASE-VER: Set to 9.1.1.
diff --git a/gcc/ChangeLog.ibm b/gcc/ChangeLog.ibm
new file mode 100644
index 00000000000..d44898dd41b
--- /dev/null
+++ b/gcc/ChangeLog.ibm
@@ -0,0 +1,4 @@
+2019-05-03   Michael Meissner  <meissner@linux.ibm.com>
+
+	Clone branch subversion id 270579
+
diff --git a/gcc/REVISION b/gcc/REVISION
new file mode 100644
index 00000000000..589b9e2462d
--- /dev/null
+++ b/gcc/REVISION
@@ -0,0 +1 @@
+[ibm/gcc-9-branch revision 270862]
diff --git a/gcc/c-family/ChangeLog.ibm b/gcc/c-family/ChangeLog.ibm
new file mode 100644
index 00000000000..d44898dd41b
--- /dev/null
+++ b/gcc/c-family/ChangeLog.ibm
@@ -0,0 +1,4 @@
+2019-05-03   Michael Meissner  <meissner@linux.ibm.com>
+
+	Clone branch subversion id 270579
+
diff --git a/gcc/c/ChangeLog.ibm b/gcc/c/ChangeLog.ibm
new file mode 100644
index 00000000000..d44898dd41b
--- /dev/null
+++ b/gcc/c/ChangeLog.ibm
@@ -0,0 +1,4 @@
+2019-05-03   Michael Meissner  <meissner@linux.ibm.com>
+
+	Clone branch subversion id 270579
+
diff --git a/gcc/testsuite/ChangeLog.ibm b/gcc/testsuite/ChangeLog.ibm
new file mode 100644
index 00000000000..d44898dd41b
--- /dev/null
+++ b/gcc/testsuite/ChangeLog.ibm
@@ -0,0 +1,4 @@
+2019-05-03   Michael Meissner  <meissner@linux.ibm.com>
+
+	Clone branch subversion id 270579
+
diff --git a/gcc/tree-ssa-pre.c b/gcc/tree-ssa-pre.c
index 7bb2cf1e220..e1c75f8e906 100644
--- a/gcc/tree-ssa-pre.c
+++ b/gcc/tree-ssa-pre.c
@@ -4197,6 +4197,7 @@ pass_pre::execute (function *fun)
   loop_optimizer_init (LOOPS_NORMAL);
   split_critical_edges ();
   scev_initialize ();
+  calculate_dominance_info (CDI_DOMINATORS);
 
   run_rpo_vn (VN_WALK);
 
diff --git a/libgcc/ChangeLog.ibm b/libgcc/ChangeLog.ibm
new file mode 100644
index 00000000000..d44898dd41b
--- /dev/null
+++ b/libgcc/ChangeLog.ibm
@@ -0,0 +1,4 @@
+2019-05-03   Michael Meissner  <meissner@linux.ibm.com>
+
+	Clone branch subversion id 270579
+
diff --git a/libgo/go/os/user/user_test.go b/libgo/go/os/user/user_test.go
index 2563077eb2e..eeb24dd0e00 100644
--- a/libgo/go/os/user/user_test.go
+++ b/libgo/go/os/user/user_test.go
@@ -129,6 +129,9 @@ func TestLookupGroup(t *testing.T) {
 
 func TestGroupIds(t *testing.T) {
 	checkGroup(t)
+	if runtime.GOOS == "aix" {
+		t.Skip("skipping GroupIds, see golang.org/issue/30563")
+	}
 	if runtime.GOOS == "solaris" {
 		t.Skip("skipping GroupIds, see golang.org/issue/14709")
 	}
